src/test/java/dev/openfeature/sdk/StructureTest.java

```diff
@@ -98,6 +98,20 @@ void mapToStructureTest() {
         assertEquals(new Value(Instant.ofEpochSecond(0)), res.getValue("Instant"));
         assertEquals(new HashMap<>(), res.getValue("Map").asStructure().asMap());
         assertEquals(new Value(immutableContext), res.getValue("ImmutableContext"));
-        assertNull(res.getValue("nullKey"));
+        assertEquals(new Value((String)null), res.getValue("nullKey"));
```

Comment:
Modified assertion associated with [this](https://github.com/open-feature/java-sdk/pull/663/files#r1370486094) change.